BibBuddy/app/src/main/res/values/strings.xml

```diff
@@ -137,16 +137,26 @@
     <string name="format_options">Format Options</string>
     <string name="format_arrow">Format Arrow</string>
 
-
     <!-- Export Dialog Alert Window Descriptions -->
     <string name="storage_permission_needed">Zugriff erforderlich</string>
     <string name="storage_permission_alert_msg">
         Zugriff auf den Gerätespeicher erforderlich,
         um Dateien aus dieser App zu exportieren.
+        Erlaube den Zugriff und versuche nochmal die
+        Dateien zu exportieren.
     </string>
-    <string name="storage_permission_ok_btn">OK</string>
+    <string name="ok">OK</string>
     <string name="storage_permission_cancel_btn">Abbrechen</string>
     <string name="storage_permission_denied">Zugriff verweigert</string>
+    <string name="file_write_failed">File write failed:</string>
+    <string name="exported_file_stored_in">Exportierte Datei ist gespeichert unter:</string>
+
+    <!-- Export: Empty Booklist -->
+    <string name="empty_booklist">Die Bücherliste ist leer</string>
```

Comment:
As suggested above, we should use consistent wording here, so this should become: "Das Regal ist leer."

Reply:
In this case, I would then also change the name of the string to "empty_shelf".

Reply:
Sure, go ahead :)